docs-src/cloud/operating-envelope-regions.md

```diff
@@ -0,0 +1,15 @@
+---
+id: operating-envelope-regions
+title: Where is Temporal Cloud hosted and running?
+sidebar_label: Regions
+description: Temporal Cloud currently runs in 10 regions in AWS.
+tags:
+  - operations
+  - temporal-cloud
+  - explanation
+---
+
+Temporal Cloud currently runs in 10 regions in Amazon Web Services (AWS).
+Although Temporal Cloud currently runs only on AWS, it works with applications running in any cloud or data center.
```

Comment:
```suggestion
Temporal Cloud currently runs in 10 regions in Amazon Web Services (AWS):

- ap-northeast-1
- ap-southeast-1
- ap-southeast-2
- ca-central-1
- eu-central-1
- eu-west-1
- eu-west-2
- us-east-1
- us-east-2
- us-west-2

Although Temporal Cloud currently runs only on AWS, it works with applications running in any cloud or data center.
```

couldn't find this info anywhere on our site or in our docs. this seems like the right place for it.